pycoin/services/biteasy.py

```diff
@@ -10,16 +10,54 @@
 from pycoin.tx.script import tools
 
 
+def fetch_json(url_extension):
+    URL = "https://api.biteasy.com/blockchain/v1/%s" % url_extension
+    r = Request(URL,
+                headers={
+                    "content-type": "application/json",
+                    "accept": "*/*",
+                    "User-Agent": "curl/7.29.0"})
+    d = urlopen(r).read()
+    json_response = json.loads(d.decode("utf8"))
+
+    return json_response
+
+
+def balance_for_address(bitcoin_address, confidence=1):
+    """
```

Comment:
I think "confirmations" might be better than "confidence" since the notion of "confidence" is a bit squishy.